programs/mmm/src/ext_util.rs

```diff
@@ -0,0 +1,120 @@
+use anchor_lang::prelude::*;
```

Comment:
can we create a util mod and put this file in it?

Reply:
The util file exists, I can merge these two